docs/examples.rst

```diff
@@ -0,0 +1,183 @@
+Examples
+########
+
+Flow Production Tracking Jira Bridge comes with several examples than can be found in
+the `examples <https://github.com/shotgunsoftware/sg-jira-bridge/tree/master/examples>`_ folder.
+
+These examples can be loaded if the `examples` directory is added to the PYTHONPATH in the
+`settings.py <https://github.com/shotgunsoftware/sg-jira-bridge/blob/master/settings.py#L95>`_ file.
+
+To properly use your custom syncer, you need to make sure you're correctly referencing it.
+This is done by using its name in the ``Jira Sync URL`` field of your Flow Production Tracking Project
+as well as in the Webhook URL on the Jira side.
+
+For example, this is how a syncer is defined in the `settings.py` file and how the urls should look like
+
+.. code-block:: python
+
+    SYNC = {
+        # Add the test syncer to the list of syncers, it will be available
+        # with the http://<your server>/jira2sg/test and http://<your server>/sg2jira/test
+        # urls.
+        "test": {
+            # Example of a custom syncer with an additional parameter to define
+            # a log level.
+            "syncer": "example_sync.ExampleSync",
+            "settings": {
+                "log_level": logging.DEBUG
+            },
+        }
+    }
+
+
+TimeLog Syncer
+**************
+
+Description
+===========
+
+The TimeLog syncer allows the synchronization of Flow Production Tracking TimeLogs with Jira Worklogs.
+It is working both way: from Flow Production Tracking to Jira and vice-versa.
+It also handles TimeLog/Worklog deletion.
+
+TimeLog Creation
+----------------
+
+When a TimeLog is created in Flow Production Tracking, an associated worklog will
+be created in Jira.
+
+.. note::
+    Only the Timelogs associated to a synced Task will be pushed to Jira. If a TimeLog is created
+    in Flow Production Tracking but is linked to a Task which don't have the ``Sync In Jira`` field checked,
+    no Jira Worklog will be created.
+
+.. warning::
+    Due to Jira API limitation, it is not possible right now to set the Worklog assignee.
+    In order to keep a track of the PTR TimeLog assignee, they are stored as JSON data in a
+    custom field on the Jira Issue.
```

Comment:
Should we be mentioning that it may be possible to achieve assignment with other Jira timetracking plugins?

Reply:
Maybe link to the "Jira API Limitations & Workaround" section?

Reply:
Link added :)